core/android_make.go

```diff
@@ -271,7 +291,7 @@ func (m *library) GenerateBuildAction(sb *strings.Builder, bt binType, ctx bluep
 		sb.WriteString("LOCAL_MODULE_OWNER := " + m.Properties.Owner + "\n")
 		sb.WriteString("LOCAL_PROPRIETARY_MODULE := true\n")
 	}
-	if m.strip() {
+	if strlib, ok := mod.(stripable); ok && strlib.strip() {
```

Comment:
Don't we expect all libraries to be stripable? We know this is definitely a binary, shared lib or static lib from the switch at the start of the function.

If we write this like:
```
if mod.(stripable).strip() {
}
```
Then we'll get a panic if we accidentally break libraries implementing `stripable`. At the moment, they would silently stop being stripped :)

Reply:
At the moment we do, but that's not necessarily the case. Static libraries don't have to implement stripable. They do at the moment (and return false) as I was having problems making it work for both android and linux. I think I can now remove library.strip()